data/hai/hai reveal 4 middle.txt

```diff
@@ -685,20 +683,22 @@ mission "Hai Reveal: Pirate Troubles [4]"
 
 
 
-mission "Hai Reveal [A04] Report to Teeneep"
+# This mission needs an odd name so it lexically sorts after "Hai Reveal: Pirate Troubles [4]"
+# Otherwise, you'll get the Teeneep note about Scar's Legion being resolved,
+# before the Scar's Legion issue is resolved
+mission "ZZ Hai Reveal [A04] Report to Teeneep"
 	landing
 	name "Report to Teeneep"
 	description `Tell Teeneep about the experience with Scar's Legion.`
-	source Hai-home
+	source "Hai-home"
 	destination Darkwaste
 	to offer
-		has "event: alondo symbol timer"
 		or
 			has "Hai Reveal: Pirate Troubles [4]: done"
 			has "Hai Reveal: Pirate Troubles [4]: declined"
+			has "hr defeated scar's legion"
 	on offer
-		dialog `Alondo is apparently away somewhere and unavailable for a short while. So, now that the situation with Scar's Legion is resolved, it's time to report to Teeneep about the experience. Maybe it will shed some light on what's going on?`
-
+		dialog `Now that the situation with Scar's Legion is resolved, it's time to report to Teeneep about the experience. Maybe it will shed some light on what's going on?`
```

Comment:
There is a missing branch in the missions where you "check in with Alondo" instead of Teeneep. You'll get the message to visit Alondo, which you can't do. If things trigger in the wrong order, you're unable to go through the missions.

Reply:
Actually, scratch that. That problem is unrelated to this line. I'll revert this while I'm testing the timer trigger.

Reply:
I reverted this change.